content/en/platform/corda/5.1/deploying-operating/deployment/deploying/manual-bootstrapping.md

```diff
@@ -82,101 +97,100 @@ corda-cli.cmd topic -b kafka-1.example.com -k config.properties create -r 3 -p 1
-{{% tab name="PowerShell" %}}
-```shell
-corda-cli.cmd topic -b <BOOTSTRAP-SERVERS> -k config.properties `
-  create -r <REPLICAS> -p <PARTITIONS> `
-  -u crypto=<CRYPTO_USER> -u db=<DB_USER> -u flow=<FLOW_USER> -u membership=<MEMBERSHIP_USER> `
-  -u p2pGateway=<P2P_GATEWAY_USER> -u p2pLinkManager=<P2P_LINK_MANAGER_USER> -u rest=<REST_USER> `
-  connect
-```
-{{% /tab %}}
-{{< /tabs >}}
+Alternatively, the Corda CLI can generate a preview of the required Kafka topic configuration in YAML. You can save, and if required modify, this content before using the Corda CLI to execute it, as follows:
 
-For information about the Corda CLI `topic` command's arguments, see the [Corda CLI reference]({{< relref"../../../reference/corda-cli/topic.md">}}).
-### Topic Creation by Scripting
+1. Use the `preview` sub-command of the Corda CLI `create` sub-command to generate a preview of the configuration:
 
-Alternatively, the Corda CLI can generate a script which you should review before executing against the broker.
-The script makes use of the `kafka-topic.sh` script provided with a Kafka installation.
+   {{< tabs name="">}}
+   {{% tab name="Bash" %}}
+   ```sh
+   corda-cli.sh topic -b <BOOTSTRAP-SERVERS> -k <CLIENT-PROPERTIES-FILE> create -r <REPLICAS> -p <PARTITIONS> preview
+   ```
+   {{% /tab %}}
+   {{% tab name="PowerShell" %}}
+   ```shell
+   corda-cli.cmd topic -b <BOOTSTRAP-SERVERS> -k <CLIENT-PROPERTIES-FILE> create -r <REPLICAS> -p <PARTITIONS> preview
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
 
-Run the following Corda CLI command to generate the script:
+   For example:
 
-{{< tabs name="cli-script">}}
-{{% tab name="Bash" %}}
-```sh
-corda-cli.sh topic -b <BOOTSTRAP-SERVERS> -k config.properties \
-  create -r <REPLICAS> -p <PARTITIONS> script -f <FILE> -c <CONCURRENCY>
-```
-{{% /tab %}}
-{{% tab name="PowerShell" %}}
-```shell
-corda-cli.cmd topic -b <BOOTSTRAP-SERVERS> -k config.properties create -r <REPLICAS> -p <PARTITIONS> script -f <FILE> -c <CONCURRENCY>
-```
-{{% /tab %}}
-{{< /tabs >}}
+   {{< tabs name="">}}
+   {{% tab name="Bash" %}}
+   ```sh
+   corda-cli.sh topic -b kafka-1.example.com -k config.properties create -p 1 -r 1 -u preview
+   ```
+   {{% /tab %}}
+   {{% tab name="PowerShell" %}}
+   ```shell
+   corda-cli.cmd topic -b kafka-1.example.com -k config.properties create -p 1 -r 1 -u preview
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
 
-Where `<FILE>` is the name of the file in which to save the script and `<CONCURRENCY>` is the number of topics to create in parallel to speed execution.
+2. Review the output, make any necessary changes, and save the configuration to a file.
+3. Use the `connect` sub-command with the `-f` argument to connect to the Kafka broker and execute the configuration:
 
-For example:
+   {{< tabs name="create-topics">}}
+   {{% tab name="Bash" %}}
+   ```sh
+   corda-cli.sh topic -b <BOOTSTRAP-SERVERS> -k <CLIENT-PROPERTIES-FILE> -f <YAML-TOPIC-FILE> create -r <REPLICAS> -p <PARTITIONS> connect
```

Comment:
The -f (--file) argument belongs to the `connect` sub-command. I think it should follow it in the example. 
```suggestion
   corda-cli.sh topic -b <BOOTSTRAP-SERVERS> -k <CLIENT-PROPERTIES-FILE> create -r <REPLICAS> -p <PARTITIONS> connect -f <YAML-TOPIC-FILE>
```